composer.json

```diff
@@ -16,13 +16,13 @@
 
     "require": {
         "php":                           ">=5.3.1",
-        "behat/mink":                    "~1.6@dev",
+        "behat/mink":                    "~1.7@dev",
```

Comment:
I removed the requirement entirely, as this driver actually delegates almost everything to BrowserKitDriver, and especially the things related to the access to Mink.